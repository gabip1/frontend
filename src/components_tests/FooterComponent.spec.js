
import React from 'react';
import renderer from 'react-test-renderer';
import FooterComponent from '../components/FooterComponent';

describe('FooterComponent', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<FooterComponent></FooterComponent>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
})
