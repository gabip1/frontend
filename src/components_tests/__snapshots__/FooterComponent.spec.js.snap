// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateMovieController renders correctly 1`] = `
<div>
  <br />
  <br />
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="card col-md-6 offset-md-3 offset-md-3"
      >
        <h3
          className="text-center"
        >
          Add movie
        </h3>
        <div
          className="card-body"
        >
          <form>
            <div
              className="form-group"
            >
              <label>
                 Title 
              </label>
              <input
                className="form-control"
                name="title"
                onChange={[Function]}
                placeholder="Title"
                value=""
              />
            </div>
            <div
              className="form-group"
            >
              <label>
                 Director 
              </label>
              <input
                className="form-control"
                name="director"
                onChange={[Function]}
                placeholder="Director"
                value=""
              />
            </div>
            <div
              className="form-group"
            >
              <label>
                Genre 
              </label>
              <input
                className="form-control"
                name="genre"
                onChange={[Function]}
                placeholder="Genre"
                value=""
              />
            </div>
            <button
              className="btn btn-success"
              onClick={[Function]}
            >
              Save
            </button>
            <button
              className="btn btn-danger"
              onClick={[Function]}
              style={
                Object {
                  "marginLeft": "10px",
                }
              }
            >
              Cancel
            </button>
          </form>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`FooterComponent renders correctly 1`] = `
<div>
  <footer
    className="footer"
  >
    <span
      className="text-muted"
    >
      All rights reserved 2021
    </span>
  </footer>
</div>
`;
